revenue_app/tests.py

```diff
@@ -234,11 +234,11 @@ def test_transactions(self, kwargs, expected_length):
         self.assertEqual(len(organizer_transactions), expected_length)
 
     @parameterized.expand([
-        ('66220941', 5, 3500),
-        ('98415193', 6, 3402),
-        ('17471621', 4, 2250),
-        ('35210860', 5, 39),
-        ('88128252', 7, 481),
+        ('66220941', 5, 17500),
+        ('98415193', 6, 20412),
+        ('17471621', 4, 9000),
+        ('35210860', 5, 195),
+        ('88128252', 7, 3367),
```

Comment:
question, why these values changed?

Reply:
The values were wrong because they were not adding the total quantity for each event.